app/models/piece.rb

```diff
@@ -1,4 +1,39 @@
 class Piece < ActiveRecord::Base
   belongs_to :game
   belongs_to :player
+
+  def obstructing_pieces(x_destination, y_destination)
```

Comment:
I would call this `get_obstructing_pieces`. Typically should follow a `verb_adverb_noun_prounoun` type order